crates/driver/src/infra/simulator/tenderly/mod.rs

```diff
@@ -102,6 +102,7 @@ pub struct Simulation {
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Linter was complaining and I don't know if I should remove it.

Reply:
Will open a PR to address this.